controllers/ControllerComment.php

```diff
@@ -2,100 +2,100 @@
+			header("Location: http://".$this->getServer()."/Auth/login");
 			exit();
 		}else{
 
 			$userId = $key['id'];
-			$comment = htmlentities(trim($this->_params['message']));
+			$comment = htmlentities(trim($this->getPost()['message']));
 			$postId = (new Session)->getKey('post');
 
-			$good = $this->_commentManager->addComment($userId, $comment, $postId['id']);
+			$good = (new CommentsManager())->addComment($userId, $comment, $postId['id']);
 
 			if ($good){
-				header("Location: http://".$this->_server."/Post/".$postId['id']);
+				header("Location: http://".$this->getServer()."/Post/viewPost/".$postId['id']);
 			}else{
 				echo 'Une erreur c\'est produite veuillez recommencer !' ;
 			}
 		}
 	}
 
-	private function updateComment(){
+	/**
+	 * Modification d'un commentaire en BDD.
+	 *
+	 * Renvoi sur la page du post concerné.
+	 */
+	public function updateComment(): void {
 
-		$key = (new Session)->getKey('post');
-		$commentId = $this->_method[3];
-		$post = $this->_params;
+		$keyPost = (new Session)->getKey('post');
+		$keyUser = (new Session)->getKey('user');
+		$post = $this->getPost();
 
 		if (isset($post['message'])){
-			$this->checkComment($post['message'], $commentId, $key['id']);
+			$this->checkComment($post['message'], $this->getSlug(), $keyPost['id']);
 		}
 
+		$content = (new CommentsManager())->oneComment((int) $this->getSlug());
 
-
-		$content = $this->_commentManager->oneComment((int) $commentId);
-
-		echo $this->_twig->render( 'comment.twig', ['logged'=> TRUE,
-		                                            'server' => $this->_server,
-		                                            'id' => $content['id'],
-		                                            'content'=> html_entity_decode($content['content'])]);
+		echo $this->getTwig()->render( 'comment.twig', ['logged'=> TRUE,
+		                                                'server' => $this->getServer(),
+		                                                'id' => $content['id'],
+		                                                'admin' => $keyUser['admin'],
+		                                                'user' => $keyUser['pseudo'],
+		                                                'content'=> html_entity_decode($content['content'])]);
 
 	}
 
-	public function checkComment(string $comment, int $commentId, int $postId){
+	/**
+	 *Vérification des informations $_POST et enregistrement du commentaire.
+	 *
+	 * @param string $comment
+	 * @param int $commentId
+	 * @param int $postId
+	 */
+	public function checkComment(string $comment, int $commentId, int $postId): void {
 
 		$content = htmlentities(trim($comment), ENT_QUOTES);
-		$update = $this->_commentManager->updateComment($content, $commentId);
+		$update = (new CommentsManager())->updateComment($content, $commentId);
 
 		if ($update){
-			header('location: http://'.$this->_server.'/Post/'.$postId);
+			header('location: http://'.$this->getServer().'/Post/viewPost/'.$postId);
```

Comment:
Codacy found an issue: [The use of function header() is discouraged](https://app.codacy.com/manual/PatOpen/OC_Blog/pullRequest?prid=6061935)